depthai_ros_driver/include/depthai_ros_driver/device_ros.hpp

```diff
@@ -293,12 +297,11 @@ class DeviceROS : public dai::DeviceBase {
                 case dai::DatatypeEnum::Buffer:
                     break;
                 case dai::DatatypeEnum::CameraControl:
-                    // @TODO: make writer to xLinkIn to work
-                    // _streams[name] =
-                    //     std::make_unique<dai::XLinkStream>(*conn, name, dai::XLINK_USB_BUFFER_MAX_SIZE);
-                    // _sub[name] = _sub_nh.subscribe(name, 1000,
-                    //         generate_cb_lambda<depthai_datatype_msgs::RawCameraControl,
-                    //                 dai::DatatypeEnum::CameraControl>(_streams[name], _sbuf, _writer_buf));
+                    _serial_bufs[name] = std::vector<std::uint8_t>();
+                    _writer_bufs[name] = std::vector<std::uint8_t>();
+                    _streams[name] =
+                        std::make_unique<dai::XLinkStream>(*conn, name, dai::XLINK_USB_BUFFER_MAX_SIZE);
+                    _sub[name] = _sub_nh.subscribe(name, 1000, generate_cb_lambda<depthai_datatype_msgs::RawCameraControl>(name));
```

Comment:
That's a huge queue size. Perhaps parametrize it

Reply:
Just a question for getting parametes. 
Currently, `rr:DeviceROS` is initialized with `rr::DeviceROS driver(p.getOpenVINOVersion());` and the constructor `DeviceROS(ros::NodeHandle nh, Args... args)` is not called. Is the constructor skipping codes inside this constructor supported or do we suppose the constructor is always used?

Reply:
We can switch to the ctor with the nodehandle.